extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -321,70 +269,42 @@
   display: block !important;
 }
```

Comment:
We might be able to remove this.

Reply:
I think this helps for keyboard users (part of the a11y efforts), that it shows the pause button for them to `tab` through and select, right?